frontend/stories/inputs/select/FormMultiSelect.stories.tsx

```diff
@@ -68,54 +68,54 @@ const Controlled: ComponentStory<typeof FormMultiSelect> = ({ formError, ...args
 				formError={formError ? { message: formError } : null}
 				{...args}
 			>
-				<MultiSelectMenuItem value={{ id: 1 }} key={1}>
+				<MultiSelectMenuItem value="1" key={1}>
 					<IconContainer>
 						<CalendarIcon />
 					</IconContainer>
 					calendar icon
 				</MultiSelectMenuItem>
-				<MultiSelectMenuItem value={{ r: 3 }} key={2}>
+				<MultiSelectMenuItem value="2" key={2}>
 					<IconContainer>
 						<ClearIcon />
 					</IconContainer>
 					clear icon
 				</MultiSelectMenuItem>
-				<MultiSelectMenuItem value={3} key={3}>
+				<MultiSelectMenuItem value="3" key={3}>
 					<IconContainer>
 						<PrintIcon />
 					</IconContainer>
 					print icon
 				</MultiSelectMenuItem>
-				<MultiSelectMenuItem value={4} key={4}>
+				<MultiSelectMenuItem value="4" key={4}>
 					<IconContainer>
 						<ChevronIcon />
 					</IconContainer>
 					chevron icon
 				</MultiSelectMenuItem>
-				<MultiSelectMenuItem value={10} key={10} disabled>
+				<MultiSelectMenuItem value="5" key={5} disabled>
 					<IconContainer>
 						<ChevronIcon />
 					</IconContainer>
 					(disabled) chevron icon
 				</MultiSelectMenuItem>
-				<MultiSelectMenuItem value={11} key={11} disabled>
+				<MultiSelectMenuItem value="6" key={6} disabled>
 					<IconContainer>
 						<ChevronIcon />
 					</IconContainer>
 					(disabled) chevron icon
 				</MultiSelectMenuItem>
-				<MultiSelectMenuItem value="number" key={5}>
-					5
-				</MultiSelectMenuItem>
-				<MultiSelectMenuItem value="number" key={6}>
-					6
-				</MultiSelectMenuItem>
-				<MultiSelectMenuItem value="number" key={7}>
+				<MultiSelectMenuItem value="7" key={7}>
```

Comment:
I found a weird bug. In the search box type a value  (e.g. 7) and delete. Repeat this a few times and the disabled values duplicate every time.
![image](https://user-images.githubusercontent.com/62481250/196972948-a8f69afc-3d6f-4158-8c24-63bcc56aaab2.png)

Then when you click out of it the tooltip appears where you click.
![image](https://user-images.githubusercontent.com/62481250/196972715-cfc7e6a9-3bb4-413d-86d0-1d6c451dd983.png)

Reply:
This might be because they share the same key value?